eagleproject/core/views.py

```diff
@@ -881,6 +881,40 @@ def subscribe(request):
     else:
         return render(request, 'subscription.html', {'form': SubscriberForm()})
 
+@csrf_exempt
+def add_subscription(request):
+    # Pre-flight request
+    if request.method == 'OPTIONS':
```

Comment:
Are we doing the same thing in all places? I think we don't have to manually do this everywhere, there's a config or modifier for this, let me check and get back in a bit

Reply:
Yeah lmk if there is... I figured there would be a standard way to do it in django. But as far as I could tell there aren't any other POST requests hitting this server from an external domain.